### PR TITLE
loading indicator when add data is not yet ready

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Backend Daten fehlen im "Add Data" Dialog](https://trello.com/c/VQ1PaPsu/1054-backend-daten-fehlen-im-add-data-dialog)

File: projects/gis/src/app/main/main.component.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
     <geoengine-layer-list
         class="mat-elevation-z4"
         [height]="middleContainerHeight$ | async | numberSanitizer"
-        [addAFirstLayerSidenavConfig]="addDataConfig | async | valueDefault: {component: AddDataComponent}"
+        [addAFirstLayerSidenavConfig]="addDataConfig | async | valueDefault: undefined"
```
Comment (michaelmattig) [inline projects/gis/src/app/main/main.component.html:36]: This has to be done in the gfbio app.component.html as well

Comment (ChristianBeilschmidt) [reply projects/gis/src/app/main/main.component.html:36]: [02ecf14](https://github.com/geo-engine/geoengine-ui/pull/496/commits/02ecf1496c92c6f7739bbf6a7621bdb6dbe14078)

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**